build/crds/build-crds.sh

```diff
@@ -98,7 +106,7 @@ build_helm_resources() {
     echo "{{- else }}"
 
     # add footer
-    cat "$CEPH_CRDS_BEFORE_1_16_FILE_PATH"
+    cat "${SCRIPT_ROOT}/cluster/examples/kubernetes/ceph/pre-k8s-1.16/crds.yaml"
```

Comment:
Why not use a new var with `DESTINATION_ROOT`?

Reply:
This isn't a file that is put into the destination. It is a file used as input from the source tree.